src/contentscript/injector.ts

```diff
@@ -350,9 +346,13 @@ export class Injector {
 
     const swarmGatewayUrl = await getSwarmGateway()
     const preferedOverlayStorage = await getPreferedOverlayStorage()
-
+    modules.sort((a, b) => {
+      if (a.manifest.type === ModuleTypes.Library) return -1
+      if (b.manifest.type === ModuleTypes.Library) return 1
+      return 0
+    })
```

Comment:
It could break module loading. Topological sorting is already implemented in src/background/utils/moduleManager.ts => resolveDependencies.